tests/config/loader/ini/test_ini_loader.py

```diff
@@ -56,3 +57,24 @@ def test_ini_loader_strip_comments(mk_ini_conf: Callable[[str], ConfigParser], c
     loader = IniLoader("tox", mk_ini_conf(f"[tox]\na={case}"), [], core_prefix="tox")
     outcome = loader.load(key="a", of_type=str, conf=None, env_name=None, chain=[], kwargs={})
     assert outcome == result
+
+
+@pytest.mark.parametrize(
+    ("env", "result"),
+    [
+        ("py35", "python -m coverage html -d cov"),
+        ("py36", "python -m coverage html -d cov\n--show-contexts"),
+    ],
+)
+def test_ini_loader_raw_with_factors(
```

Comment:
Feels to me this is a factor specific test, so should be https://github.com/tox-dev/tox/blob/rewrite/tests/config/loader/ini/test_factor.py